stdlib/oauth2/src/main/ballerina/src/oauth2/inbound_oauth2_provider.bal

```diff
@@ -43,9 +50,16 @@ public type InboundOAuth2Provider object {
         if (credential == "") {
             return false;
         }
-        boolean authenticated = false;
-        string username = "";
-        string scopes = "";
+
+        var oauth2Cache = self.oauth2Cache;
+        if (oauth2Cache is cache:Cache && oauth2Cache.hasKey(credential)) {
+            var oauth2CacheEntry = authenticateFromCache(oauth2Cache, credential);
+            if (oauth2CacheEntry is InboundOAuth2CacheEntry) {
+                auth:setAuthenticationContext("oauth2", credential);
+                auth:setPrincipal(oauth2CacheEntry.username, oauth2CacheEntry.username, getScopes(oauth2CacheEntry.scopes));
```

Comment:
Let's break this line to accommodate 120 line break.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/20607/commits/ac2e33994b83d3fe692cde951572b14345838ea9